external/storm-hdfs/src/main/java/org/apache/storm/hdfs/spout/Configs.java

```diff
@@ -33,79 +34,93 @@
     public static final String TEXT = "text";
     public static final String SEQ = "seq";
     /**
+     * HDFS URI.
```

Comment:
Consider moving the comments on the fields into the Javadoc. E.g. here you could put "Required - HDFS name node".